zipline/data/loader.py

```diff
@@ -162,19 +163,22 @@ def load_market_data(trading_day=None, trading_days=None, bm_symbol='^GSPC'):
         # We need the trading_day to figure out the close prior to the first
         # date so that we can compute returns for the first date.
         trading_day,
+        environ,
     )
     tc = ensure_treasury_data(
         bm_symbol,
         first_date,
         last_date,
         now,
+        environ,
     )
     benchmark_returns = br[br.index.slice_indexer(first_date, last_date)]
     treasury_curves = tc[tc.index.slice_indexer(first_date, last_date)]
     return benchmark_returns, treasury_curves
 
 
-def ensure_benchmark_data(symbol, first_date, last_date, now, trading_day):
+def ensure_benchmark_data(symbol, first_date, last_date, now, trading_day,
+                          environ=None):
```

Comment:
Yea, the call site is [here](https://github.com/quantopian/zipline/pull/1793/files#diff-0ec9e6870d90568b4a83e0daa3a317b1R166).